### PR TITLE
44 add setup safe event

### PR DESCRIPTION
**Pull request by JacqueGM**
- [x]  add safesetup event from 1.3.0
- [x] add latest version of GnosisSafe and Proxy factory contracts
- [x] test in dockers with latest contracts
- [x] refactor the code in `src/gnosisSafe.ts` so that we recycle code
- [x] update circles-contracts package
- [x] test in production

### CODE REVIEW

**Review by llunaCreixent — COMMENTED** [review]
Looks great! just remember to delete the logging code

File: src/utils.ts [hunk 1]
```diff
@@ -25,3 +26,23 @@ export function createBalanceID(token: Address, holder: Address): string {
   return token.toHexString().concat('-').concat(holder.toHexString())
 }
 
+export function manageOwnership(owner: string, safe: string, transaction: string, blockNumber: BigInt, logIndex: BigInt, blocktimestap: BigInt): void {
```
Comment (llunaCreixent) [inline src/utils.ts]: owner and safe can be type Address

**Review by llunaCreixent — APPROVED** [review]
yay!

**Comment by JacqueGM** [comment]
Closes #44

**Pull request merged**